src/pe_reports/metrics.py

```diff
@@ -133,7 +137,7 @@ def summary(self):
                     "name_server",
                 ]
             ]
-            domain_sum = domain_sum[:25]
+            domain_sum.loc[domain_sum["ipv6"] == "", "ipv6"] = "NA"
```

Comment:
It's not clear to me what is happening in this line.  Can you add a comment that explains?

Reply:
Added comment in this commit:[d456379](https://github.com/cisagov/pe-reports/pull/565/commits/d45637968b769748453b6999e457719a96525757)